karma.conf.js

```diff
@@ -1,5 +1,7 @@
-const browsers = require('./src/browsers');
-const webpackConfig = require('./webpack.config');
+const cwd = process.cwd();
```

Comment:
out of curiosity, were there path related issues happening or is this just for consistency?